src/resume.json

```diff
@@ -39,30 +39,73 @@
   ],
 
   "skills": [
-    {
-      "name": "Web Development",
-      "keywords": [
-        "HTML",
-        "CSS",
-        "CSS Animations and Transitions",
-        "Responsive Design",
-        "Javascript",
-        "ESlint and Prettier",
-        "GitFlow",
-        "SEO",
-        "React"
-      ]
-    }
+    [
+      {
+        "name": "HTML",
+        "level": "Intermediate",
+        "logo": "html.png"
+      }
+    ],
+    [
+      {
+        "name": "CSS",
+        "level": "Intermediate",
+        "logo": "css.png"
+      }
+    ],
+    [{ "name": "Javascript", "level": "Intermediate", "logo": "js.png" }],
+    [{ "name": "React", "level": "Intermediate", "logo": "react.png" }],
+    [
+      {
+        "name": "CSS Animations and Transitions",
```

Comment:
It is just a personal opinion but I feel text looks prettier if only the first character is capital and rest small.
`CSS Animations and Transitions` -> `CSS animations and transitions`